build.gradle

```diff
@@ -310,8 +311,13 @@ artifactory {
 
 dependencies {
     implementation "com.synopsys.integration:integration-rest:10.3.6"
-    implementation 'com.synopsys.integration:hub-imageinspector-lib:14.3.5'
+    implementation 'com.synopsys.integration:hub-imageinspector-lib:14.3.7'
     implementation 'ch.qos.logback:logback-classic:1.2.13'
+    constraints {
+        implementation('ch.qos.logback:logback-core:1.2.13') {
+            because 'previous version has a vulnerability'
+        }
+    }
```

Comment:
Is my understanding correct that `constraints` is being used to force even the transitive dependencies to use 1.2.13 or higher only?

Reply:
Just 1.2.13 to match the classic version.